app/src/main/java/org/mozilla/tv/firefox/ext/WebBackForwardList.kt

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.ext
+
+import android.webkit.WebBackForwardList
+
+/**
+ * Convert WebBackForwardList to List<String> so it is easier to use.
+ */
+fun WebBackForwardList.toStringList(): List<String> {
```

Comment:
nit: `toUrlStringList` or similar: we want to be explicit that we're extracting the URLs.

However, I think I'd prefer to leave it as, `WebBackForwardList.toList()` and the callers can extract the string if they want: it's more flexible this way.